site/docs/3.4.4/index.html

```diff
@@ -0,0 +1,28 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more
+ contributor license agreements.  See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<!DOCTYPE html>
+<html lang="en-US">
+<meta charset="utf-8">
+<title>Redirecting&hellip;</title>
+<link rel="canonical" href="https://archive.apache.org/dist/spark/docs/3.4.4/">
+<script>location="https://archive.apache.org/dist/spark/docs/3.4.4/"</script>
+<meta http-equiv="refresh" content="0; url=https://archive.apache.org/dist/spark/docs/3.4.4/">
```

Comment:
The document is intentionally in `archive.apache.org` because this is a newly created link for the EOL release.